build.gradle.kts

```diff
@@ -403,6 +403,8 @@ allprojects {
                     disable(
                         "ComplexBooleanConstant",
                         "EqualsGetClass",
+                        "InlineMeSuggester",
+                        "JavaUtilDate",
```

Comment:
Currently the code used ` @SuppressWarnings("JdkObsolete")`, so I would rather suggest replacing it with `@SuppressWarnings("JavaUtilDate")`.
Then we would know if when usages of Date appear.

Reply:
Will submit a patch in a second, and then look into replacing the formatters with newer ones.